zproject/test_settings.py

```diff
@@ -173,6 +173,15 @@ def set_loglevel(logger_name: str, level: str) -> None:
 SOCIAL_AUTH_GOOGLE_KEY = "key"
 SOCIAL_AUTH_GOOGLE_SECRET = "secret"
 SOCIAL_AUTH_SUBDOMAIN = 'auth'
+SOCIAL_AUTH_APPLE_SERVICES_ID = 'com.zulip.chat'
+SOCIAL_AUTH_APPLE_BUNDLE_ID = 'com.zulip.bundle.id'
+SOCIAL_AUTH_APPLE_CLIENT = 'com.zulip.chat'
+SOCIAL_AUTH_APPLE_KEY = 'KEYISKEY'
+SOCIAL_AUTH_APPLE_TEAM = 'TEAMSTRING'
+SOCIAL_AUTH_APPLE_SECRET = get_from_file_if_exists("zerver/tests/fixtures/apple/private_key.pem")
```

Comment:
Can we add a comment, ideally in the file itself, but worst case here, making clear how this private key was generated and why it's safe to have it checked into version control?

Reply:
Added a comment in the file with the command used to generate the file, the reason we needed it and saying it's safe to have it checked into version control.